controllers.js

```diff
@@ -26,4 +27,27 @@ exports.getTopics= (req,res,next) =>{
 }
 
 
+exports.getArticleId= (req,res,next) =>{
+    const {article_id}= req.params
+    if(!article_id || isNaN(article_id)){
+        res.status(400).send({msg:'Invalid point'})}
```

Comment:
If it's not a number, psql will handle it for you ("22P02"). Use psql error handler rather than creating a customs error